src/sentry/web/urls.py

```diff
@@ -366,7 +369,7 @@ def init_all_applications():
     # Keep named URL for for things using reverse
     url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_id>[\w_-]+)/issues/(?P<group_id>\d+)/$', react_page_view,
         name='sentry-group'),
-    url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_id>[\w_-]+)/$', react_page_view,
+    url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_slug>[\w_-]+)/$', ReactProjectPageView.as_view(),
```

Comment:
We'd also need to make sure anything else that tries to reverse `sentry-stream` is updated as well.